src/Model/ImageLoading.py

```diff
@@ -220,7 +220,7 @@ def get_roi_info(dataset_rtss):
     return dict_roi
 
 
-def calc_dvhs(dataset_rtss, dataset_rtdose, rois, interrupt_flag, dose_limit=None):
+def calc_dvhs(dataset_rtss, dataset_rtdose, rois, interrupt_flag, dose_limit=None, progress_callback=None):
```

Comment:
failed to update the docstring to add this optional parameter.
Knowing the type of the progress callback would be useful